todo/src/components/Todo.js

```diff
@@ -1,11 +1,26 @@
-import React from 'react';
-
-const Todo = props => {
-  return (
-    <div onClick={() => props.toggleTodo(props.todo.id)} className={props.todo.completed ? `cleared` : ``}>
-      <li>{props.todo.task}</li>
-    </div>
-  )
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { toggleTodo } from '../actions';
+
+class Todo extends Component {
+
+  toggleTodo = id => {
+    this.props.toggleTodo(id);
+  }
+
+  render() {
+    return (
+      <div onClick={() => this.toggleTodo(this.props.todo.id)} className={this.props.todo.completed ? `cleared` : ``}>
+        <li>{this.props.todo.task}</li>
+      </div>
+    )
+  }
 }
 
-export default Todo;
+const mapStateToProps = (state) => {
+    return {
+        ...state
+    };
+};
```

Comment:
note that since you already have all the info you need being passed in from TodoList on props, you don't need to access anything else from the store here. in this case it'd be fine to not use a MSTP at all